ros/src/common/libs/amathutils_lib/test/src/test_kalman_filter.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2018-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <ros/ros.h>
+#include <tf/transform_datatypes.h>
+
+#include "amathutils_lib/kalman_filter.hpp"
+#include "amathutils_lib/kalman_filter_delayed_measurement.hpp"
+
+class KalmanFilterTestSuite : public ::testing::Test
+{
+  public:
+    KalmanFilterTestSuite() {}
+    ~KalmanFilterTestSuite() {}
+};
+
+TEST_F(KalmanFilterTestSuite, updateCase)
+{
+
+    KalmanFilter kf;
+    Eigen::MatrixXd x = Eigen::MatrixXd::Zero(3, 1);
+    Eigen::MatrixXd y = Eigen::MatrixXd::Zero(3, 1);
+    Eigen::MatrixXd P = Eigen::MatrixXd::Identity(3, 3);
+    Eigen::MatrixXd A = Eigen::MatrixXd::Zero(3, 3);
+    Eigen::MatrixXd B = Eigen::MatrixXd::Zero(3, 1);
+    Eigen::MatrixXd C = Eigen::MatrixXd::Identity(3, 3);
+    Eigen::MatrixXd R = Eigen::MatrixXd::Identity(3, 3);
+    Eigen::MatrixXd y_pred = y;
+
+    ASSERT_EQ(false, kf.update(y)) << "uninitialized, should be return false";
```

Comment:
what about changing from `should be return false` to `false expected`?

Reply:
modified following your suggestion in a62ee25.